src/media/inner.rs

```diff
@@ -276,8 +291,8 @@ impl MediaInner {
         // We don't actually want this buffer here, but it must be created before we
         // get to the do_packetize() (as part of handle_timeout).
         let _ = self.buffers_tx.entry(pt).or_insert_with(|| {
-            let max_retain = if spec.codec.is_audio() { 4096 } else { 2048 };
-            PacketizingBuffer::new(spec.codec.into(), max_retain)
+            let max_retain = if is_audio { 4096 } else { 2048 };
```

Comment:
unrelated to this pr, but why is audio 4096 and else is 2048?

Reply:
NACK handling. Maybe a tad big :)